core/src/test/java/com/linecorp/armeria/internal/annotation/AnnotatedHttpServiceFactoryTest.java

```diff
@@ -122,6 +132,35 @@ public void ofUserDefinedRepeatableDecorator() throws NoSuchMethodException {
         assertThat(udd2.value()).isEqualTo(2);
     }
 
+    @Test
+    public void testFindAnnotatedServiceElementsWithPathPrefixAnnotation() {
+        final Object object = new PathPrefixServiceObject();
+        final List<AnnotatedHttpServiceElement> elements = find("/", object, new ArrayList<>());
+
+        final List<String> paths = elements.stream()
+                                           .map(AnnotatedHttpServiceElement::route)
+                                           .map(Route::paths)
```

Comment:
This returns the list which has two same paths for one `route`. How about just getting the first element here?
```java
final List<String> paths = elements.stream()
                                   .map(AnnotatedHttpServiceElement::route)
                                   .map(route -> route.paths().get(0))
                                   .collect(Collectors.toList());
assertThat(paths).containsExactly("/home/hello", "/home/");
...